homepage/models.py

```diff
@@ -40,25 +44,47 @@ class Req_Code(models.IntegerChoices):
     req_code = models.SmallIntegerField(choices=Req_Code.choices, default=Req_Code.NONE)
 
 
-class User(models.Model):
-    name = models.CharField(max_length=100)
-    email = models.CharField(max_length=50)
-    password = models.CharField(max_length=30)
+class AppUser(models.Model):
+    user = models.OneToOneField(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, primary_key=True)
+
+    def __str__(self):
+        return self.user.username
 
 
 class Review(models.Model):
     course = models.ForeignKey(Course, on_delete=models.CASCADE)
-    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    user = models.ForeignKey(AppUser, on_delete=models.CASCADE)
     rate = models.SmallIntegerField()
     date = models.DateTimeField(default=timezone.now)
     content = models.TextField()
     course_load = models.SmallIntegerField()
 
+    def __str__(self):
+        review_asList = (self.content).split()
+        shortened_review = ""
+        MAX_WORDS_PREVIEW = 5
+        wordsCounter = 1
+
+        for word in review_asList:
+            shortened_review = shortened_review + word + ' '
+            wordsCounter += 1
+
+            if(wordsCounter > MAX_WORDS_PREVIEW):
+                break
+
+        return 'Preview: {}'.format(shortened_review)
+
 
 class Professor(models.Model):
     name = models.CharField(max_length=100)
 
+    def __str__(self):
+        return self.name
+
 
 class Professor_to_Course(models.Model):
     professor_id = models.ForeignKey(Professor, on_delete=models.CASCADE)
     course_id = models.ForeignKey(Course, on_delete=models.CASCADE)
+
+    def __str__(self):
+        return 'p = {}, c = {}'.format(self.professor_id, self.course_id)

```

Comment:
I think prof and cour will be more appropriate names than p and c

Reply:
agree

Reply:
Changed.